clang/test/Frontend/skip-function-bodies.cc

```diff
@@ -0,0 +1,8 @@
+// Trivial check to ensure skip-function-bodies flag is propagated.
+//
+// RUN: %clang_cc1 -verify -skip-function-bodies -pedantic-errors %s
```

Comment:
Could you also check in the test that this flag is not available outside of `-cc1`?

I think we really want to avoid that flag in the user-facing interface as it is expected to break a lot of things.